core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dicom-core"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
I advice to let the owner or member of the crate handle versioning, this is more simple for the owner, since cargo package are not really link to git, the owner can merge several pull request and make the bump when wanted.

Reply:
That would be right, but in this case the upstream branch already has dicom-core at "0.4.1". This is probably some artifact from merging.